projects/sturdy/index.js

```diff
@@ -0,0 +1,50 @@
+const { request, gql } = require("graphql-request");
+const {toUSDTBalances} = require("../helper/balances");
+
+const graphUrl =
+    'https://api.thegraph.com/subgraphs/name/sturdyfi/sturdy-fantom';
+
+const graphQuery = gql`
+    query get_tvl {
+        reserves {
+            name
+            symbol
+            decimals
+            price {
+                priceInEth
+            }
+            totalDeposits
+            totalLiquidity
+            availableLiquidity
+        }
+    }
+`;
+
+const priceInUSD = (value, decimals, price) => {
+    return (value / Math.pow(10, decimals) * price / Math.pow(10, 8)).toFixed(2);
+}
+
+async function fetch(borrow) {
+    const { reserves } = await request(graphUrl, graphQuery);
+    const tvl = reserves.reduce((sum, reserve) => {
+        const value = borrow ? reserve.totalLiquidity - reserve.availableLiquidity : reserve.totalDeposits;
```

Comment:
can this be changed to ... : reserves.availableLiquidity;

Reply:
this should be `reserve.totalDeposits;`

Reply:
we exclude borrows from core tvl, so it should be supply - borrows. Users can filter on the borrows on our UI

Reply:
Thank you for your guidance.
I changed the calculation logic and pushed it. 
Please review once more.